src/asm2cfg/asm2cfg.py

```diff
@@ -134,8 +139,16 @@ def get_jump_pattern(stripped, function_name):
     '0x000055555555600f:        jmp  0x55555555603d'
     """
     if stripped:
-        return re.compile(r'0x0*([01-9a-fA-F]+):\W+\w+\W+0x0*([01-9a-fA-F]+)$')
-    return re.compile(fr'<\+(\d+)>:.+<{function_name}\+(\d+)>')
+        return re.compile(fr'0x0*({hex_pattern}):\W+\w+\W+0x0*({hex_pattern})$')
+    return re.compile(fr'<[+-](\d+)>:.+<{function_name}[+-](\d+)>')
+
+
+def get_conditional_branch_pattern(stripped):
+    """
+    Return regexp pattern used to identify conditional jumps.
+    """
+    # TODO: implement this
+    return re.compile(r'')
```

Comment:
Could you add conditional jump support in a separate PR if you mean to add support for this? This way the repo stays more clean from TODOs :)

Reply:
Right, I'll move this to dedicated PR.